code/modules/mob/living/living_defense.dm

```diff
@@ -405,6 +405,18 @@
 	..()
 	setMovetype(movement_type & ~FLOATING) // If we were without gravity, the bouncing animation got stopped, so we make sure we restart the bouncing after the next movement.
 
+/mob/living/proc/do_slap_animation(atom/A)
```

Comment:
Please document this proc to meet our [code documentation guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-proc) for contributors.